src/gc.c

```diff
@@ -61,11 +61,7 @@ GarbageCollectorCtx *NewGarbageCollector(const RedisModuleString *k, float initi
   GarbageCollectorCtx *gc = malloc(sizeof(*gc));
 
   *gc = (GarbageCollectorCtx){
-      .timer = NULL,
-      .hz = initialHZ,
-      .keyName = k,
-      .stats = {},
-      .rdbPossiblyLoading = 1,
+      .timer = NULL, .hz = initialHZ, .keyName = k, .stats = {}, .rdbPossiblyLoading = 1,
```

Comment:
Formatting: you just changed that to multiple newlines in the previous commit, no? Personally, I prefer multiline.

Reply:
@itamarhaber we're trying to use a formatter file for universal formatting - i don't like auto-formatters, but it's the easiest to work with.

Reply:
just installed a clang plugin on the eclipse so it should be fine from now on